ci/jenkins/mellanox/README.md

```diff
@@ -0,0 +1,56 @@
+# Mellanox CI
+
+Mellanox CI is used to test hardware offload in the Antrea project. It deploys Kubernetes with the Antrea image and tests whether the OVS has hardware offload capabilities or not.
```

Comment:
Could you wrap the lines at 80 characters in documentation files?

Reply:
Done for normal lines, but faced some issues with the table.
It seems it is not possible to split lines for markdown table cells, for now i left them as is, do you think we need to handle it?

Reply:
Maybe you can use "<br />" to split a long line. Now comments column can be hardly seen...
https://stackoverflow.com/questions/11700487/newline-in-markdown-table